src/ai/backend/storage/dellemc/onefs_client.py

```diff
@@ -200,35 +199,28 @@ async def create_quota(
             "thresholds_on": "fslogicalsize",
             "enforced": True,
         }
-        headers = {
-            "Content-Type": "application/json",
-            "Accept": "application/hal+json",
-        }
         async with self._request(
             "POST",
             "quota/quotas",
-            headers=headers,
-            data=data,
+            json=data,
         ) as resp:
             msg = await resp.json()
         return msg
 
-    async def delete_quota(self, quota_id):
+    async def delete_quota(self, quota_id) -> None:
         async with self._request(
             "DELETE",
             f"quota/quotas/{quota_id}",
-        ) as resp:
-            msg = await resp.json()
-        return msg
+        ) as _:
+            return
```

Comment:
Wouldn't it be better to add a `resp.raise_for_status()` guard to halt consequent progresses when OneFS API fails?

Reply:
Don't need it because the `raise_for_status` parameter is set as `True` in `_request()` method